endpoints/openrtb2/auction.go

```diff
@@ -251,6 +251,9 @@ func (deps *endpointDeps) Auction(w http.ResponseWriter, r *http.Request, _ http
 	}
 
 	labels, ao = sendAuctionResponse(w, hookExecutor, response, req.BidRequest, account, labels, ao)
+	if len(ao.Errors) == 0 {
+		exchange.RecordResponsePreparationMetrics(auctionRequest.MakeBidsTimeInfo, deps.metricsEngine)
+	}
```

Comment:
Nitpick: Is probably better to move `RecordResponsePreparationMetrics` definition to this package

Reply:
done as part of [67acb2e](https://github.com/prebid/prebid-server/pull/2658/commits/67acb2ec761bb10ba84c3fbfd7014a127c7c6b68)

@guscarreon @VeronikaSolovei9  can you re-review. Previous review was dismissed by new changes